js/viz/gauges/utils/resolve_label_overlapping.js

```diff
@@ -0,0 +1,115 @@
+const ARC_COORD_PREC = 5;
+
+import { normalizeAngle, normalizeArcParams } from '../../core/utils';
+
+export function dividePoints(bars) {
+    return bars.reduce(function(stackBars, bar) {
+        const angle = normalizeAngle(bar._angle);
+        const isRightSide = angle <= 90 || angle >= 270;
+        bar._text._lastCoords = { x: 0, y: 0 };
+
+        (isRightSide ? stackBars.right : stackBars.left)
+            .push({
+                series: {
+                    isStackedSeries: () => false,
+                    isFullStackedSeries: () => false
+                },
+                getLabels: () => [{
+                    isVisible: () => true,
+                    getBoundingRect: () => {
+                        const { height, width, x, y } = bar._text.getBBox();
+                        const lastCoords = bar._text._lastCoords;
+
+                        return {
+                            x: x + lastCoords.x,
+                            y: y + lastCoords.y,
+                            width,
+                            height,
+                        };
+                    },
+                    shift: (x, y) => {
+                        const box = bar._text.getBBox();
+
+                        bar._text._lastCoords = { x: x - box.x, y: y - box.y };
+                        bar._text.attr({ translateX: x - box.x, translateY: y - box.y });
+                    },
+                    draw: ()=> bar.hideLabel(),
```

Comment:
still strange style of arrow functions)
```suggestion
                    draw: () => bar.hideLabel(),
```

Reply:
An interesting point
Of course I'll fix it, but I'm wondering why lint didn't work automatically

Reply:
Fixed